pkg/kn/commands/version/version_test.go

```diff
@@ -16,31 +16,27 @@ package version
 
 import (
 	"bytes"
+	"encoding/json"
 	"testing"
 	"text/template"
 
-	"knative.dev/client/pkg/kn/commands"
-
 	"github.com/spf13/cobra"
 	"gotest.tools/assert"
-)
+	"sigs.k8s.io/yaml"
 
-type versionOutput struct {
-	Version     string
-	BuildDate   string
-	GitRevision string
-}
+	"knative.dev/client/pkg/kn/commands"
+)
 
 var versionOutputTemplate = `Version:      {{.Version}}
 Build Date:   {{.BuildDate}}
 Git Revision: {{.GitRevision}}
 Supported APIs:
 * Serving
-  - serving.knative.dev/v1 (knative-serving v0.13.0)
+  - {{ index .SupportedAPIs.serving 0}}
 * Eventing
-  - sources.eventing.knative.dev/v1alpha1 (knative-eventing v0.13.1)
-  - sources.eventing.knative.dev/v1alpha2 (knative-eventing v0.13.1)
-  - eventing.knative.dev/v1alpha1 (knative-eventing v0.13.1)
+  - {{ index .SupportedAPIs.eventing 0}}
```

Comment:
👍 maybe we could use a gotemplate loop here ?

Reply:
done.